class AuthException implements Exception{
  final String mensagem;

  const AuthException(this.mensagem);
}